unliftio/ChangeLog.md

```diff
@@ -1,7 +1,8 @@
-## 0.1.1.0
+## 0.2.0.0
```

Comment:
Adding a new module isn't a breaking change. Could you make the version `0.1.2.0` instead?